tests/test_config_consistency.py

```diff
@@ -0,0 +1,73 @@
+# Unless explicitly stated otherwise all files in this repository are licensed under the the Apache License Version 2.0.
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2022 Datadog, Inc.
+
+from utils import weblog, interfaces, scenarios, features
+
+
+@scenarios.default
+@features.tracing_configuration_consistency
+class Test_ConfigServerErrorStatusesDefault:
```

Comment:
Should the naming convention be `Test_Config_<test-specifics>_<condition>`?
E.g, `Test_Config_ServerErrorStatuses_Default` and `Test_Config_ServerErrorStatuses_FeatureFlag`
And, Test_Config_ServiceMapping_Default` and `Test_Config_ServiceMapping_FeatureFlag`
So `Test_Config_` is a prefix that represents the grouping of all the tests we're writing for config consistency ?

Reply:
Yeah this is a great idea, done in ed91063a0ba031e35928a28d0e59806faec7730b